From 67b11a4f224c97001234df4fceaa8cc8bbae3063 Mon Sep 17 00:00:00 2001
From: Anders Berg <anders.berg@lsi.com>
Date: Mon, 20 Jan 2014 16:52:32 +0100
Subject: [PATCH 35/71] arm/axxia: Configure changes to 'GPIO driver for SPI
 chip selects'

git://git.yoctoproject.org/linux-yocto-3.10 standard/axxia/base
commit 6f7e00f0e87d85d7d7cb5e924b49e171872b0754

This patch was split off as configuration changes to
support the GPIO driver changes.

Signed-off-by: Anders Berg <anders.berg@lsi.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/boot/dts/axm5504-sim.dts      |    4 ++++
 arch/arm/boot/dts/axm5516-amarillo.dts |    4 ++++
 arch/arm/boot/dts/axm55xx.dtsi         |    6 ++++++
 3 files changed, 14 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/axm5504-sim.dts b/arch/arm/boot/dts/axm5504-sim.dts
index 2dd964b..7580ae5 100644
--- a/arch/arm/boot/dts/axm5504-sim.dts
+++ b/arch/arm/boot/dts/axm5504-sim.dts
@@ -106,6 +106,10 @@
 	status = "okay";
 };
 
+&spics {
+	status = "okay";
+};
+
 &spi0 {
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/axm5516-amarillo.dts b/arch/arm/boot/dts/axm5516-amarillo.dts
index 85c4ede..a802b52 100644
--- a/arch/arm/boot/dts/axm5516-amarillo.dts
+++ b/arch/arm/boot/dts/axm5516-amarillo.dts
@@ -78,6 +78,10 @@
 	status = "okay";
 };
 
+&spics {
+	status = "okay";
+};
+
 &spi0 {
 	status = "okay";
 
diff --git a/arch/arm/boot/dts/axm55xx.dtsi b/arch/arm/boot/dts/axm55xx.dtsi
index fc81a75..f774531 100644
--- a/arch/arm/boot/dts/axm55xx.dtsi
+++ b/arch/arm/boot/dts/axm55xx.dtsi
@@ -370,6 +370,12 @@
 				#size-cells = <0>;
 				reg = <0x20 0x10088000 0x00 0x1000>;
 				interrupts = <0 42 4>;
+				num-cs = <5>;
+				cs-gpios = <&spics 0 1>,
+					   <&spics 1 1>,
+					   <&spics 2 1>,
+					   <&spics 3 1>,
+					   <&spics 4 1>;
 				status = "disabled";
 			};
 		};
-- 
1.7.5.4

